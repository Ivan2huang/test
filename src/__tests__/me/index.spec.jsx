import React from 'react';
import { render } from '@testing-library/react';

import MePage from '../../pages/me';

jest.mock('../../layouts/withLayoutProvider', () => Component => () => (
  <div>
    <header>Layout Header</header>
    <Component />
  </div>
));

jest.mock('../../modules/me', () => ({
  Me: () => <div>Me Component</div>,
}));

describe('Me Index Page', () => {
  it.skip('should match snapshot', () => {
    const { container, debug } = render(<MePage />);
    debug();
    expect(container).toMatchSnapshot();
  });
});
