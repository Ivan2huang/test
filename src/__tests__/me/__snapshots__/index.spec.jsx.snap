// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Me Index Page should match snapshot 1`] = `
<div>
  <div>
    <header>
      Layout Header
    </header>
    <div
      class="MuiBox-root MuiBox-root-2"
    >
      <div
        class="MuiGrid-root MuiGrid-container"
      />
    </div>
  </div>
</div>
`;
